services/graph/pkg/service/v0/driveitems.go

```diff
@@ -469,6 +469,10 @@ func (g Graph) Invite(w http.ResponseWriter, r *http.Request) {
 
 	condition := unifiedrole.UnifiedRoleConditionGrantee
 	if IsSpaceRoot(statResponse.GetInfo().GetId()) {
+		if statResponse.GetInfo().GetSpace().GetSpaceType() == _spaceTypePersonal {
```

Comment:
I think this check should live in the shareprovider (https://github.com/cs3org/reva/blob/edge/internal/grpc/services/usershareprovider/usershareprovider.go#L159)

Reply:
@rhafer Moved to reva. Please review [https://github.com/cs3org/reva/pull/4559](https://github.com/cs3org/reva/pull/4559)